src/pages/core/entrypoints/reply.mdx

```diff
@@ -0,0 +1,28 @@
+---
+tags: ["core", "entrypoints"]
+---
+
+import { Callout } from "nextra/components";
+
+# Reply
```

Comment:
I think this section could be extended with a description of how the reply is requested (`Response::add_submessage`) and how the IDs work.

Reply:
Added a little example on how it is used. Should hopefully be good enough? Not sure.